scripts/src/mock_client.ts

```diff
@@ -81,10 +151,41 @@ class Client {
 		return {
 			headers: {
 				"x-request-id": uuid4(),
-				"Authorization": `Bearer ${this.token}`,
+				"Authorization": this.authToken ? `Bearer ${this.authToken.token}` : "",
 			},
 		};
 	}
+
+	private createWallet(): string {
+		this.keyPair = StellarSdk.Keypair.random();
+		return this.keyPair.publicKey();
+	}
+
+	private async stellarOperation(operation: StellarSdk.AbstractOperation): Promise<StellarSdk.TransactionResult> {
+		const accountResponse = await STELLAR.server.loadAccount(this.keyPair.publicKey());
+		const transaction = new StellarSdk.TransactionBuilder(accountResponse).addOperation(operation).build();
+
+		transaction.sign(this.keyPair); // sign the transaction
+		return await STELLAR.server.submitTransaction(transaction);
+	}
+
+	private async establishTrustLine(): Promise<StellarSdk.TransactionResult> {
+		const op = StellarSdk.Operation.changeTrust({
+			asset: STELLAR.kinAsset
+		});
+
+		for (let i = 0; i < 3; i++) {
```

Comment:
maybe we can write a retry function that encapsulates this logic and receives a callback + args